proposals/MultipleReadersWriters.md

```diff
@@ -221,6 +221,11 @@ childAccessHandle.close();
 await parentHandle.remove(); // successful
 ```
 
+### Interactions with BFCache
+A page should not be made aware that it or another page is or has been in the BFCache. A fully active page could be made aware of a BFCached page if there is contention between locks they hold. The File System Access API operations/primitives hold locks that may cause this issue.
```

Comment:
The key issue is that _a page may still hold a file system lock when it enters the BFCache_. The last sentence hints at this, but I think this paragraph could be more concise if that was stated more directly